.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I should approve the PR 
![image](https://user-images.githubusercontent.com/210499/28318306-f2d02cca-6bd2-11e7-94dd-aa2b5a4ce04e.png)
then if no conflicts, at the bottom of the PR front page you will see

![image](https://user-images.githubusercontent.com/210499/28318341-1cf176e4-6bd3-11e7-98b4-7cfa3c661256.png)

but this will be enable and then you can merge it with the master.
I use the word "merge" , but we newer really make a merges, we use rebase and merge.
rebase is a spcial command that applies the commits of your bracnch over the commits of the master and do not make a "merge commit" like this
![image](https://user-images.githubusercontent.com/210499/28318428-7d4eb5c4-6bd3-11e7-89f1-1a31c5c34f59.png)
so the commits in the master looks clear and order. 
You should check what debase is and ho to do it, also you should check and how to resolve conflicts. This tools help many team members to review and merge their daily work.